convert2rhel/utils.py

```diff
@@ -618,4 +625,44 @@ def set_locale():
     os.environ.update({"LC_ALL": "C", "LANG": "C"})
 
 
+def get_secrets_from_conf_file(path_to_cfg=None):
+    """Get the secret (activation key or password) from config file.
+
+    The order is:
+    1. if filename is set then load from it
+    2. ~/.convert2rhel.ini
+    3. /etc/convert2rhel.ini
+
+    Return {"password": <value>, "activation-key": <value>}.
+    If the secret isn't set, default value is None.
+    """
+    header = "SECRET"  # header in config file
+    secret_names = ("password", "activation-key")
+    config = configparser.ConfigParser()
+    paths = CONFIG_PATHS
+    secrets = {"password": None, "activation-key": None}
+
+    if path_to_cfg:
+        paths.insert(0, path_to_cfg)  # highest priority
+
+    for path in paths:
+        if os.path.exists(path):
+            config.read(path)
+        else:
+            continue
+
+        for secret_name in secret_names:
+            try:
+                secrets[secret_name] = config.get(header, secret_name)
+                loggerinst.debug("Found %s in %s" % (secret_name, path))
+
+            except (configparser.NoOptionError, KeyError, configparser.NoSectionError):
+                continue
```

Comment:
I feel this for-loop is a bit unnecessary here. We can access the fields in the config file manually since we are using only 2 fields now, and we don't need to optimize our way in right now (Maybe when we start porting more options to the config file it could be beneficial) 
```suggestion
          try:
              secrets["password"] = config.get("SECRETS",  "password")
              secrets["activation-key"] = config.get("SECRETS",  "activation-key")
          except (configparser.NoOptionError, KeyError, configparser.NoSectionError):
              # Maybe throwing an error here is a better option?
              loggerinst.critical("Failed to load config file options.")
              
          loggerinst.debug("Loaded the config file successfully!")
```